source/StartConditions.h

```diff
@@ -15,32 +15,53 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include "Account.h"
 #include "ConditionSet.h"
+#include "Conversation.h"
 #include "Date.h"
+#include "Ship.h"
 
 #include <list>
 
 class DataNode;
 class Planet;
-class Ship;
+class Sprite;
 class System;
 
 
 
 class StartConditions {
 public:
+	StartConditions() = default;
+	
+	// Implicit constructor
+	StartConditions(const DataNode &node);
+	
 	void Load(const DataNode &node);
 	// Finish loading the ship definitions.
 	void FinishLoading();
+	// Serialize the basic information of this start
+	void Save(DataWriter &out) const;
 	
 	Date GetDate() const;
-	// Get this start's planet, or New Boston if not set.
+	
 	const Planet &GetPlanet() const;
-	// Get this start's system, or Rutilicus if not set.
 	const System &GetSystem() const;
+	
+	// Choose the appropiate conversation to display
+	const Conversation &GetConversation() const;
+	
+	// The sprite that will be outlined on StartConditionsPanel.cpp
+	const Sprite *GetSprite() const;
+	
+	const std::string &GetName() const;
+	const std::string &GetDescription() const;
+	
 	const Account &GetAccounts() const;
 	const ConditionSet &GetConditions() const;
 	const std::list<Ship> &Ships() const;
 	
+	// Check whether a start scenario is valid
+	// A valid start scenario has a name, a date, a system and a planet
```

Comment:
Would no longer be true with my suggestion.
```suggestion
	// A valid start scenario has a valid system, planet, and conversation.
```